<launch>
  <param name="function" value="2" />
  <param name="reg_min_cluster_size" value="5000" />
  <param name="reg_max_cluster_size" value="10000" />
  <param name="reg_cthres" value="0.05" />
  <param name="reg_sthres" value="6" />
  <param name="reg_k" value="100" />
  <node pkg="perception" type="point_cloud_demo" name="point_cloud_demo" output="screen">
    <remap from="cloud_in" to="mock_point_cloud" />
  </node>
</launch>


